.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: install substreams cli
         uses: jaxxstorm/action-install-gh-release@v1.7.1
         with:
-          repo: streamingfast/substreams
+          repo: robinbernon/substreams-messari
```

Comment:
why `robinbernon/substreams-messari`?

Reply:
They have now merge my PR into their main branch so I've put this back to what it was originally. Unfortunately they have not yet made a release for the new commit to master so the build agent is still pulling down the previous version which is why build is still failing.I'll keep an eye out for when they make the new release.

Reply:
sounds good!